.github/workflows/zizmor.yaml

```
@@ -0,0 +1,37 @@
+# https://github.com/woodruffw/zizmor
+name: GitHub Actions Security Analysis with Zizmor
+
+on:
+  push:
+    branches: ["master", "main"]
```

Just for parsimony in this particular repo :slightly_smiling_face: 

```suggestion
    branches: ["main"]
```